### DISCUSSION TITLE
Using pragmatic-drag-and-drop-react-beautiful-dnd-migration inside a <table>

### DISCUSSION THREAD

**Discussion by sigh71** [atlassian/pragmatic-drag-and-drop#70] [Q&A]
Hi,

Long time users of react-beautiful-dnd we're trying to switch to pragmatic-drag-and-drop-react-beautiful-dnd-migration with an existing component that movesrows inside a <table>. When a drag starts the following is written to console:

```
react_devtools_backend_compact.js:2367 Warning: validateDOMNesting(...): <div> cannot appear as a child of <tbody>.
    at div
    at Placeholder (node_modules_atlaskit_pragmatic-drag-09040d.bundle.js:5616:24)
    at Draggable (node_modules_atlaskit_pragmatic-drag-09040d.bundle.js:5091:23)
    at tbody
    at table
    at Droppable (node_modules_atlaskit_pragmatic-drag-09040d.bundle.js:7065:23)
    at DragDropContextProvider
```

Is there any guidance on working around this?

**Comment by Dron007** [comment]
I have the same issue. I used @atlaskit/pragmatic-drag-and-drop-react-beautiful-dnd-migration but it just doesn't work as expected. The table is corrupted during dragging the row. It stops me from using this library as react-beautiful-dnd alternative. We also need to be sure that it works with React 19 and there is no clear mentions about it.
